src/components/Section/index.tsx

```diff
@@ -119,7 +119,7 @@ function Section({
                     {overlayContent?.()}
                 </View>
             )}
-            <View style={[styles.w100, isCentralPane && (isSmallScreenWidth ? styles.p5 : styles.p8)]}>
+            <View style={[styles.w100, isCentralPane && (shouldUseNarrowLayout ? styles.p5 : styles.p8)]}>
```

Comment:
```suggestion
            <View style={[styles.w100, isCentralPane && (isSmallScreenWidth ? styles.p5 : styles.p8)]}>
```

Causing

https://github.com/Expensify/App/assets/75031127/35b1143a-1b58-47e4-b8bb-c89b784538f5